browser/resources/settings/email_aliases_page/content/LoadingIcon.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { color } from '@brave/leo/tokens/css/variables'
+import * as React from 'react'
+import Icon from '@brave/leo/react/icon'
+import styled from 'styled-components'
+
+const SpinningIcon = styled(Icon)`
```

Comment:
This doesn't seem neccessary when we have `ProgressRing` in nala. https://nala.bravesoftware.com/?path=/story/components-progress-ring--default-ring

Reply:
thanks -- fixed